changelog/index.rst

```diff
@@ -0,0 +1,48 @@
+DC/OS ChangeLog
+===============
+
+
+Every PR in DC/OS is accompanied with a newsfragment file against the release.
+
+The newsfragment file will be of format
+
+::
+
+    DCOS_OSS-<id>|DCOS-<id>.{feature, bugfix, doc, removal, misc}
+
+
+* **<id>** : Refers to a JIRA id. 
+* **.feature** : Signifying a new feature.
+* **.bugfix** : Signifying a bug fix.
+* **.doc** : Signifying a documentation improvement.
+* **.removal** : Signifying a deprecation or removal of public API.
+* **.misc** : A ticket has been closed, but it is not of interest to users.` 
```

Comment:
I like this proposal, and appreciate the thought and effort involved in putting this together. It will make generating RN easier and push engineers to provide a clear, concise, accurate description of each issue they work on, which is good for clarity of thought and communication. However, I'm wondering if some engineers will push back, viewing this as extra work (even if the effort is small) and not wanting to comply with it. We should invest some effort in explaining the context and reasons for the change, so engineers view it as a benefit and not a burden.